### ISSUE TITLE
Http.request errors on python target

### ISSUE THREAD

**Issue by asciiascetic** [HaxeFoundation/haxe#5228]
Using the following simple test:

```haxe

class Get {
  public static function main () {
    trace('starting');

    var r = new haxe.Http( "http://ichibausa.com");
    r.setHeader("User-Agent","Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:46.0) Gecko/20100101 Firefox/46.0");

    trace('new request');
    
    r.onData = function (data) {
      trace(data);
    };

    r.onError = function (e) {
      trace("ERROR");
      trace(e);
    };
    
    trace('defined onData');

    r.request( false );
    trace('made request');
  }
}

```

Compiled with `haxe -main Get -python get.py` and run with  `python3 get.py`, will always return the error handler and print: 

`TypeError("unsupported operand type(s) for +: 'int' and 'builtin_function_or_method'",)`

Where are an integer and a function being added ?  Somewhere in the generated python?

**Comment by asciiascetic** [comment]
The error seems to appear within the `writeBytes` method in the generated `sys_net_Socket_SocketOutput` class.  

There is an attempt to add a variable `pos` to the `len` built in method in the line

```python
payload = data[pos:pos+len]
```
which should be

```python
payload = data[pos:pos+_hx_len]
```

instead.  I'm not sure about how to fix/patch this, any help?

**Comment by nadako** [comment]
This looks like a duplicate of #4594, fixed in efa707d6e56f34720052274386b078dd31b00343

**Comment by asciiascetic** [comment]
Ah cool, thanks.
